components/Pages/DocuSignPage/CartTotal.tsx

```diff
@@ -1,13 +1,13 @@
-import React, { useState } from 'react'
-import Button from '../Button'
-import ReadHereLink from '../ReadHereLink'
-import { useContractBuilderProvider } from '@/providers/ContractBuilderProvider'
-import { CONTRACT_BUILDER_STEP } from '@/hooks/useContractBuilder'
-import InfoDialog from '../InfoDialog'
+import React from 'react'
+
+import { useModalProvider } from '@/providers/ModalProvider'
+import Button from '@/components/Button'
+import ReadHereLink from '@/components/ReadHereLink'
+import { useRouter } from 'next/navigation'
 
 const CartTotal = () => {
-  const { setTab } = useContractBuilderProvider()
-  const [isOpen, setIsOpen] = useState(false)
+  const { setIsDocuSignModalOpen } = useModalProvider()
+  const router = useRouter()
   const MoneyPara = ({ children }: { children: React.ReactNode }) => {
```

Comment:
Consider renaming MoneyPara for clarity, as it’s used for displaying amounts but doesn’t clearly convey its purpose.